### ISSUE TITLE
Modeltranslation version 0.18.1 crashes on start

### ISSUE THREAD

**Issue by henri-hulski** [deschler/django-modeltranslation#639]
Version 0.18.1 is installing fine.
But when starting my app with runserver, modeltranlation crashes because of a missing file:
`./env/lib/python3.9/site-packages/VERSION`.

Here the traceback:
```
    __import__("modeltranslation")
  File "./env/lib/python3.9/site-packages/modeltranslation/__init__.py", line 11, in <module>
    __version__ = (Path(__file__).parent.parent / "VERSION").read_text().strip()
  File "/usr/lib/python3.9/pathlib.py", line 1255, in read_text
    with self.open(mode='r', encoding=encoding, errors=errors) as f:
  File "/usr/lib/python3.9/pathlib.py", line 1241, in open
    return io.open(self, mode, buffering, encoding, errors, newline,
  File "/usr/lib/python3.9/pathlib.py", line 1109, in _opener
    return self._accessor.open(self, flags, mode)
FileNotFoundError: [Errno 2] No such file or directory: './env/lib/python3.9/site-packages/VERSION'
```

**Comment by last-partizan** [comment]
Could you please test if it works in master?

**Comment by henri-hulski** [comment]
Yep! It works.

**Comment by last-partizan** [comment]
Great, released `0.18.2`

**Comment by henri-hulski** [comment]
Thank's for your work!
